<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MacroEditorScriptRunnerDlg</class>
 <widget class="QDialog" name="MacroEditorScriptRunnerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>487</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Script runner</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/img/test/active_script.png</normaloff>:/img/test/active_script.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblSourse">
     <property name="text">
      <string>Source</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="editSource"/>
   </item>
   <item>
    <widget class="QLabel" name="lblVariables">
     <property name="text">
      <string>Variables</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableVariables"/>
   </item>
   <item>
    <widget class="QLabel" name="lblScript">
     <property name="text">
      <string>Script</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="editScript"/>
   </item>
   <item>
    <widget class="QLabel" name="lblResult">
     <property name="text">
      <string>Result</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="editResult"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="loButtons">
     <item>
      <spacer name="spacerLeft">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnRun">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacerRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>MacroEditorScriptRunnerDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
